reports/report_builder.py

```diff
@@ -4530,6 +4530,10 @@ def tabulate_historical(self, ws, current_ws, cols, rows, c=None, r=6, write_row
                 columns = cols
                 if canon('N') in data:
                     columns = columns + [('N', 'N')]
+                if canon('n_traditional') in data:
+                    columns.insert(3, ('n_traditional', 'n_traditional'))
```

Comment:
Do we have to insert it at `3` i.e. why not `columns = columns + [...]` similar to how we handle `'N'` and `'n_digital'`?

Reply:
This is because of how it gets displayed on the sheet. `n_traditional` should be displayed after all the `TM` while `n` and ` n_digital` will always be at the end.
As shown below, the `N` after the `TM` is what we have as `n_traditional`
<img width="478" alt="Screenshot 2021-02-23 at 09 14 23" src="https://user-images.githubusercontent.com/13068580/108808451-bde07180-75b7-11eb-9739-c156b2bc55ae.png">